sys/src/9/amd64/dat.h

```diff
@@ -225,10 +224,7 @@ enum
 struct ICC
 {
 	/* fn is kept in its own cache line */
-	union{
-		void	(*fn)(void);
-		unsigned char	_ln1_[ICCLNSZ];
-	};
+	alignas(ICCLNSZ) void	(*fn)(void);
```

Comment:
This works on clang too right? This is kind of interesting trend as we move further away from ever running under Ken C again (I'm fine with that :-)) unless Ken C gets some upgrades. This is way clearer than the original as to intent ...

Reply:
It does indeed work with clang. It's actually a C11 thing.